src/flamegraph/mod.rs

```diff
@@ -232,6 +232,9 @@ pub struct Options<'a> {
     /// This is only meant to be used in tests.
     #[doc(hidden)]
     pub no_javascript: bool,
+
+    /// Diffusion-based color: the wider the frame, the brighter the color.
```

Comment:
I think it would be good to document this option more. In particular, it would be good to include what kind of uses it is useful for.